### DISCUSSION TITLE
How to install dependency track in my local meachine

### DISCUSSION THREAD

**Discussion by Ranjithkumar-Arumugam-agilysys** [DependencyTrack/dependency-track#2925] [General]
Hi,

Can you guide me how to download the dependency track in local system with this link - https://docs.dependencytrack.org/getting-started/deploy-exewar/
Where and How to install the war file in local ? 
With the above link i can able to see the link alone while i am using in cmd it won't give any thing.

**Comment by nscuro** [comment]
You can download the JARs (they no longer are `.war` files, we need to update the docs) directly from GitHub Releases: https://github.com/DependencyTrack/dependency-track/releases/tag/4.8.2

Please keep in mind that non-containerized distributions are deprecated and may be discontinued in a future release.

**Reply by Ranjithkumar-Arumugam-agilysys** [reply]
Hi nscuro,

Guide me - i have download the dependency track bundled jar can i directly installed in my local system ? you have any documentation to follow the steps for initial setup for dependency track. 

I try to download the dependency track bundled it downloaded in my local but i try to install, its not installing.
